### PR TITLE
Updated vendor contribution guidelines 

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Updated [Integrate your technology with AMP](https://amp.dev/documentation/guides-and-tutorials/contribute/integrate-your-tech/?format=websites) to new contributor guidelines. Moved relevant documentation into subfolder and updated paths on amp.dev. 

Please add anyone who needs to review :)

### CODE REVIEW

**Comment by tharders** [comment]
@CrystalOnScript Also please check the links inside the moved files. It seems some relative links where not updated. You can run 'npm run posttest' which will also check the links and report problems.

**Comment by CrystalOnScript** [comment]
@lannka friendly ping

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+There are 90+ ad providers who have added support for advertising related features like amp-ad. Here is a [sample pull request](https://github.com/ampproject/amphtml/pull/2299) from the ad network [Criteo](https://github.com/ampproject/amphtml/blob/master/ads/criteo.md).
+
+## Level two contribution 
+
+Level two vendor contributions share components. All logic is checked into the AMP repository, and no custom Javascript can be loaded into an iframe.  Vendors add configurations or features to existing extensions to implement their functionalities. If such a component does not exist they may propose a new one.
```
Comment (lannka) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md]: I saw we mix use `component` & `extension`, shall we unify?

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+### Level two examples
+
+[**amp-analytics**](../../../components/reference/amp-analytics.md)
+
+AMP analytics allows you to send events back to your server based on triggers configured by you. We have written an [analytics integration guide ](../../optimize-measure/configure-analytics/index.md)to get you started.
+
+If you only need to add a tracking pixel with dynamic parameters to your tracking URL, check out [`amp-pixel`](../../../components/reference/amp-pixel.md). Be sure to document usage on your support pages for developers that may want to use your technology with AMP.
+
+There are 20+ analytics providers who have added support to amp-analytics. Here is a [sample pull request](https://github.com/ampproject/amphtml/pull/1595) from the analytics provider [Parse.ly](https://www.parsely.com/help/integration/google-amp/).
```
Comment (lannka) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md]: I guess we don't need to mention the number? Soon the number will be out of date.

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md (hunk 1 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md]: Good catch, thanks!

**Comment by CrystalOnScript** [comment]
Sorry @sebastianbenz, need some help with this one too - it looks like it wasn't able to retrieve the error log? D:

**Comment by tharders** [comment]
@CrystalOnScript Your branch is outdated. 
The build problem was fixed in the future branch.
You have to rebase your branch against the latest future branch.
Be aware, that there is a conflict with the `ad-integration-guide.md`.
Since this file is imported you can ignore the changes in the future branch and run the import afterwards to get the latest changes.

**Comment by CrystalOnScript** [comment]
Closing out this one, because I didn't do the rebase correctly and how its [here](https://github.com/ampproject/amp.dev/pull/3175). Whoops

**Pull request closed without merging**